src/enrichment_facility.h

```diff
@@ -233,81 +265,127 @@ class EnrichmentFacility : public cyclus::Facility {
+  inline void  InitialFeed(double qty) { initial_feed = qty; }
+  inline double InitialFeed() const { return initial_feed; }
+
+  inline const cyclus::toolkit::ResBuf<cyclus::Material>& Tails() const {
+    return tails;
+  } 
 
-  inline const cyclus::toolkit::ResourceBuff& Tails() const { return tails; } 
-  
  private:
   ///   @brief adds a material into the natural uranium inventory
-  ///   @throws if the material is not the same composition as the in_recipe
+  ///   @throws if the material is not the same composition as the feed_recipe
   void AddMat_(cyclus::Material::Ptr mat);
 
-  ///   @brief generates a request for this facility given its current state. The
-  ///   quantity of the material will be equal to the remaining inventory size.
+  ///   @brief generates a request for this facility given its current state.
+  ///   Quantity of the material will be equal to remaining inventory size.
   cyclus::Material::Ptr Request_();
 
   ///  @brief Generates a material offer for a given request. The response
-  ///  composition will be comprised only of U235 and U238 at their relative ratio
-  ///  in the requested material. The response quantity will be the same as the
-  ///  requested commodity.
+  ///  composition will be comprised only of U235 and U238 at their relative
+  ///  ratio in the requested material. The response quantity will be the
+  ///  same as the requested commodity.
   ///
   ///  @param req the requested material being responded to
   cyclus::Material::Ptr Offer_(cyclus::Material::Ptr req);
 
   cyclus::Material::Ptr Enrich_(cyclus::Material::Ptr mat, double qty);
 
-  ///   @brief records and enrichment with the cyclus::Recorder
-  void RecordEnrichment_(double natural_u, double swu);
+  ///  @brief calculates the feed assay based on the unenriched inventory
+  double FeedAssay();
 
-  #pragma cyclus var {"tooltip": "input commodity", \
-                      "doc": "commodity that the enrichment facility accepts", \
-                      "uitype": "incommodity"}
-  std::string in_commod;
-  #pragma cyclus var {"tooltip": "output commodity", \
-                      "doc": "commodity that the enrichment facility supplies", \
-                      "uitype": "outcommodity"}
-  std::string out_commod;
-  #pragma cyclus var {"tooltip": "input commodity recipe", \
-                      "doc": "recipe for enrichment facility's input commodity", \
-                      "uitype": "recipe"}
-  std::string in_recipe;
-
-  #pragma cyclus var {"default": 0.03, "tooltip": "tails assay", \
-                      "doc": "tails assay from the enrichment process"}
+  ///  @brief records and enrichment with the cyclus::Recorder
+  void RecordEnrichment_(double natural_u, double swu);
+  
+  #pragma cyclus var { \
+    "tooltip": "feed",							\
+    "doc": "feed commodity that the enrichment facility accepts",	\
+    "uitype": "incommodity" \
+  }
+  std::string feed;
```

Comment:
No problem. I was thinking about this myself as I was making the first round of changes. . .